src/Forms/UrlField.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+/**
+ * Text input field with validation for a url using filter_var($url, FILTER_VALIDATE_URL)
+ * The value of the field must start with a protocol such as http:// or https://
+ */
+class UrlField extends TextField
+{
+    protected $inputType = 'text';
+
+    public function Type()
+    {
+        return 'text';
+    }
+
+    /**
+     * @param Validator $validator
+     *
+     * @return string
+     */
```

Comment:
```suggestion
```
This is already declared in the PHPDoc for this method in `FormField` - no need to redeclare the same types here, especially since the return type here is wrong.

Reply:
Removed